src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,65 @@
+import Sidebar from "../components/Sidebar";
+import "../index.css";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const ProfilePage = (): JSX.Element => {
+    return (
+        <div className="container">
+            <Sidebar />
+            <div className='some-page-wrapper'>
+                <div className="main-content">
+                    <h1 className="content-title">Profile</h1>
+                </div>
+                <div className='column outset'>
+
+                    <div className='one-column inset'>
+                        <h1 className="second-text">Personal</h1>
+                        <form>
+                            <input type="text" placeholder="First Name" className="text-input" />
```

Comment:
```suggestion
                            <input type="text" placeholder="First Name" className="text-input inset" />
```
Apply inset styling here and on all other inputs